applications/EigenSolversApplication/custom_python/add_custom_solvers_to_python.h

```diff
@@ -16,6 +16,9 @@
 #include <pybind11/pybind11.h>
 
 // External includes
+#if defined USE_FEAST4
+#define _MKL_SOLVERS_EE_H //exclude MKL's FEAST implementation
```

Comment:
could you please briefly explain why this is necessary, just for the record?

Reply:
parts of FEAST are also included in MKL, but not all of it. This define excludes the definitions of the functions for MKL in `mkl/include/mkl_solvers_ee.h`. Should I add this explanation as a comment in the code too?

Reply:
> Should I add this explanation as a comment in the code too?

yes I think this will be helpful in the future

Reply:
done